frontend/app/.server/domain/services/provincial-government-insurance-plan.service.ts

```diff
@@ -26,23 +26,23 @@ export class ProvincialGovernmentInsurancePlanServiceImpl implements ProvincialG
     this.log = logFactory.createLogger('ProvincialGovernmentInsurancePlanServiceImpl');
 
     // set moize options
-    this.findAll.options.maxAge = 1000 * this.serverConfig.LOOKUP_SVC_ALL_PROVINCIAL_GOVERNMENT_INSURANCE_PLANS_CACHE_TTL_SECONDS;
-    this.findById.options.maxAge = 1000 * this.serverConfig.LOOKUP_SVC_PROVINCIAL_GOVERNMENT_INSURANCE_PLAN_CACHE_TTL_SECONDS;
+    this.listProvincialInsurancePlans.options.maxAge = 1000 * this.serverConfig.LOOKUP_SVC_ALL_PROVINCIAL_GOVERNMENT_INSURANCE_PLANS_CACHE_TTL_SECONDS;
+    this.getProvincialInsurancePlanById.options.maxAge = 1000 * this.serverConfig.LOOKUP_SVC_PROVINCIAL_GOVERNMENT_INSURANCE_PLAN_CACHE_TTL_SECONDS;
   }
 
-  private findAllImpl(): ProvincialGovernmentInsurancePlanDto[] {
+  private listProvincialInsurancePlansImpl(): ProvincialGovernmentInsurancePlanDto[] {
     this.log.debug('Get all provincial government insurance plans');
     const provincialGovernmentInsurancePlanEntities = this.provincialGovernmentInsurancePlanRepository.findAll();
     const provincialGovernmentInsurancePlanDtos = this.provincialGovernmentInsurancePlanDtoMapper.mapProvincialGovernmentInsurancePlanEntitiesToProvincialGovernmentInsurancePlanDtos(provincialGovernmentInsurancePlanEntities);
     this.log.trace('Returning provincial government insurance plans: [%j]', provincialGovernmentInsurancePlanDtos);
     return provincialGovernmentInsurancePlanDtos;
   }
 
-  findAll = moize(this.findAllImpl, {
+  listProvincialInsurancePlans = moize(this.listProvincialInsurancePlansImpl, {
     onCacheAdd: () => this.log.info('Creating new findAll memo'),
   });
 
-  private findByIdImpl(id: string): ProvincialGovernmentInsurancePlanDto | null {
+  private getProvincialInsurancePlanByIdImpl(id: string): ProvincialGovernmentInsurancePlanDto | null {
```

Comment:
Based on our service method naming conventions, this `get` should throw an exception rather than return `null`.